hooks/post_gen_project.py

```diff
@@ -20,5 +20,4 @@ def prepare_git() -> None:
     )
 
 if __name__ == "__main__":
-    print("Executing script!!")
     prepare_git()
```

Comment:
Instead of printing we may add some logging above i.e. - 
```python
import subprocess
import logging

logging.basicConfig(level=logging.INFO)

def prepare_git() -> None:
    git_author_name = "{{cookiecutter.full_name}}"
    git_author_email = "{{cookiecutter.email}}"
    git_branch = "{{cookiecutter.branch}}"

    commands = [
        ["git", "init", "-b", git_branch],
        ["git", "config", "user.name", git_author_name],
        ["git", "config", "user.email", git_author_email]
    ]

    for command in commands:
        try:
            subprocess.run(command, check=True)
            logging.info(f"Ran command: {' '.join(command)}")
        except subprocess.CalledProcessError as e:
            logging.error(f"An error occurred while running command: {' '.join(command)}. Error: {e}")
            break

if __name__ == "__main__":
    logging.info("Preparing git...")
    prepare_git()
    logging.info("Git setup complete.")
```

Reply:
I think this file will be irrelevant once we migrate to `copier` in PR #34 as I removed the hooks directory and this file and added these procedures under `_tasks:` key in `copier.yml`. Overall copier has better support for inbuilt exception handling so we should be good with that I guess?